### ISSUE TITLE
Error on 10.11.1 build from CLI or GUI

### ISSUE THREAD

**Issue by AfterSpencer** [MagerValp/AutoDMG#131]
I am unable to build 10.11.1 from CLI or GUI

Help resolving would be appreciated.

[AutoDMG-log.txt](https://github.com/MagerValp/AutoDMG/files/26365/AutoDMG.txt)

**Comment by MagerValp** [comment]
The OS installer starts but fails for some unknown reason. Could you please check /var/log/system.log, between November 3, 2015 at 9:32:12 AM MST and  November 3, 2015 at 9:33:02 AM MST?

**Comment by AfterSpencer** [comment]
Nov  3 09:32:10 080-mavbuild-01 storagekitd[1743]: Journaling already enabled for disk: <DADisk 0x7fbf40d035f0 [0x7fff74af1390]>{id = /dev/disk4s2}
Nov  3 09:32:14 080-mavbuild-01 emond[1850]: SetUpLogs: uid = 0 gid = 0
Nov  3 09:32:14 080-mavbuild-01 emond[1850]: SetUpLogs: opening /Library/Logs/EventMonitor/EventMonitor.error.log
Nov  3 09:32:14 080-mavbuild-01 com.apple.xpc.launchd[1]: assertion failed: 15B42: launchd + 86915 [13D08717-B23A-38BE-85A6-085B051F8322]: 0x11
Nov  3 09:32:14 080-mavbuild-01 com.apple.xpc.launchd[1] (com.apple.emond): Service only ran for 0 seconds. Pushing respawn out by 10 seconds.
Nov  3 09:32:24 080-mavbuild-01 emond[1853]: SetUpLogs: uid = 0 gid = 0
Nov  3 09:32:24 080-mavbuild-01 emond[1853]: SetUpLogs: opening /Library/Logs/EventMonitor/EventMonitor.error.log
Nov  3 09:32:24 080-mavbuild-01 com.apple.xpc.launchd[1]: assertion failed: 15B42: launchd + 86915 [13D08717-B23A-38BE-85A6-085B051F8322]: 0x11
Nov  3 09:32:24 080-mavbuild-01 com.apple.xpc.launchd[1] (com.apple.emond): Service only ran for 0 seconds. Pushing respawn out by 10 seconds.
Nov  3 09:32:34 080-mavbuild-01 emond[1854]: SetUpLogs: uid = 0 gid = 0
Nov  3 09:32:34 080-mavbuild-01 emond[1854]: SetUpLogs: opening /Library/Logs/EventMonitor/EventMonitor.error.log
Nov  3 09:32:34 080-mavbuild-01 com.apple.xpc.launchd[1]: assertion failed: 15B42: launchd + 86915 [13D08717-B23A-38BE-85A6-085B051F8322]: 0x11
Nov  3 09:32:34 080-mavbuild-01 com.apple.xpc.launchd[1] (com.apple.emond): Service only ran for 0 seconds. Pushing respawn out by 10 seconds.
Nov  3 09:32:44 080-mavbuild-01 emond[1855]: SetUpLogs: uid = 0 gid = 0
Nov  3 09:32:44 080-mavbuild-01 emond[1855]: SetUpLogs: opening /Library/Logs/EventMonitor/EventMonitor.error.log
Nov  3 09:32:44 080-mavbuild-01 com.apple.xpc.launchd[1]: assertion failed: 15B42: launchd + 86915 [13D08717-B23A-38BE-85A6-085B051F8322]: 0x11
Nov  3 09:32:44 080-mavbuild-01 com.apple.xpc.launchd[1] (com.apple.emond): Service only ran for 0 seconds. Pushing respawn out by 10 seconds.
Nov  3 09:32:54 080-mavbuild-01 emond[1856]: SetUpLogs: uid = 0 gid = 0
Nov  3 09:32:54 080-mavbuild-01 emond[1856]: SetUpLogs: opening /Library/Logs/EventMonitor/EventMonitor.error.log
Nov  3 09:32:54 080-mavbuild-01 com.apple.xpc.launchd[1]: assertion failed: 15B42: launchd + 86915 [13D08717-B23A-38BE-85A6-085B051F8322]: 0x11
Nov  3 09:32:54 080-mavbuild-01 com.apple.xpc.launchd[1] (com.apple.emond): Service only ran for 0 seconds. Pushing respawn out by 10 seconds.
Nov  3 09:33:02 080-mavbuild-01 python[1754]: Socket at /tmp/se.gu.it.IEDSocketListener.2b11b07e7dd79772 failed: [Errno 40] Message too long
Nov  3 09:33:02 --- last message repeated 2 times ---
Nov  3 09:33:02 080-mavbuild-01 AutoDMG[1684]: Workflow failed: Build failed (OSInstall.mpkg failed with return code 1)
Nov  3 09:33:02 080-mavbuild-01 AutoDMG[1684]: Build failed: Build failed
Nov  3 09:33:02 080-mavbuild-01 AutoDMG[1684]:     OSInstall.mpkg failed with return code 1
Nov  3 09:33:02 080-mavbuild-01 storagekitd[1743]: Ejecting 1 disks
Nov  3 09:33:02 080-mavbuild-01 storagekitd[1743]: Removing client connection <SKDaemonConnection: 0x7fbf40c18c80>
Nov  3 09:33:02 080-mavbuild-01 storagekitd[1743]: Ejecting SKDisk { BSD Name: disk5s1 Mount point: /Volumes/OS X Base System Role: kSKDiskRoleLegacyMacData Type: kSKDiskTypeHFS }
Nov  3 09:33:02 080-mavbuild-01 storagekitd[1743]: No more connections, storagekitd will exit...
Nov  3 09:33:02 080-mavbuild-01 mdworker[1852]: (ImportBailout.Error:1325) Asked to exit for Diskarb
Nov  3 09:33:02 080-mavbuild-01 kernel[0]: hfs: unmount initiated on OS X Base System on device disk5s1
Nov  3 09:33:02 080-mavbuild-01 deleted[368]: normalizeUserMountpoint:738 volRoot failed for /Volumes/OS X Base System
Nov  3 09:33:02 080-mavbuild-01 AutoDMG[1684]: PyObjC: Converting exception to Objective-C:
Nov  3 09:33:02 --- last message repeated 1 time ---
Nov  3 09:33:02 080-mavbuild-01 storagekitd[1743]: Ejected
Nov  3 09:33:03 080-mavbuild-01 deleted[368]: __38-[CacheDelete updateInFlight:set:add:]_block_invoke:293 trying to add "com.apple.activity_tracing.cache-delete", which is already inFlight
Nov  3 09:33:03 080-mavbuild-01 kernel[0]: hfs: unmount initiated on Digital Camera RAW Compatibility Update 6.17 on device disk2s2
Nov  3 09:33:03 080-mavbuild-01 kernel[0]: hfs: unmount initiated on iTunes on device disk3s2

**Comment by MagerValp** [comment]
(I changed your paste to an attached file, it makes the thread a little easier to follow :)

This pops out:
`Nov 3 09:33:02 080-mavbuild-01 python[1754]: Socket at /tmp/se.gu.it.IEDSocketListener.2b11b07e7dd79772 failed: [Errno 40] Message too long`

Could you please try this build: https://www.dropbox.com/s/hunp7xnrh56k5xz/AutoDMG-1.5.3b407.dmg

I added some extra debug logging, so please save system.log from when it runs.

**Comment by AfterSpencer** [comment]
I get this message when I try to upload the log files:

Attaching documents requires write permission to this repository.  Choose Files Try again with a PNG, GIF, or JPG.

https://dl.dropboxusercontent.com/u/39210/build.txt
https://dl.dropboxusercontent.com/u/39210/system.txt

**Comment by MagerValp** [comment]
Oh, how annoying. I wonder if it's a setting I can control. Anyway, thanks for the logs, it's clear that I have a bug in my code that blocks the error message from being displayed. The actual error is:

```
Error Domain=PKInstallErrorDomain
Code=110 \"An error occurred while extracting files from the package \U201cEssentials.pkg\U201d.\"
UserInfo={
	NSLocalizedDescription=An error occurred while extracting files from the package \U201cEssentials.pkg\U201d.,
	NSFilePath=/Volumes/Macintosh HD 1/.OSInstallSandboxPath/Root,
	NSURL=file:///tmp/dmg.YZN4LA/Packages/Essentials.pkg,
	PKInstallPackageIdentifier=com.apple.pkg.Essentials,
	NSUnderlyingError=0x7fe2b608b1e0 {
		Error Domain=PKXARArchiveErrorDomain
		Code=101 \"archive verify failed\"
		UserInfo={
			NSURL=file:///tmp/dmg.YZN4LA/Packages/Essentials.pkg#Payload,
			NSFileOwnerAccountID=0,
			NSFileHFSTypeCode=0,
			NSFileSystemFileNumber=350,
			NSFileExtensionHidden=false,
			NSFileSystemNumber=16777223,
			NSFileSize=5552278969,
			NSFileGroupOwnerAccountID=0,
			NSFileOwnerAccountName=root,
			NSFilePosixPermissions=420,
			NSFileHFSCreatorCode=0,
			NSFileCreationDate=2015-10-18 03:31:02 +0000,
			NSFileType=NSFileTypeRegular,
			NSFileGroupOwnerAccountName=wheel,
			NSFileReferenceCount=1,
			NSFileModificationDate=2015-10-18 03:32:11 +0000,
			NSLocalizedDescription=archive verify failed
		}
	}
}
```
I think it looks like your OS X installer is corrupt.

**Comment by AfterSpencer** [comment]
I will re-download the installer and give it a try.

**Comment by AfterSpencer** [comment]
Good call that seems to have been the issue. Thanks for the help.

**Comment by MagerValp** [comment]
Good to hear!

**Comment by MagerValp** [comment]
Message too long error fixed in 28c2b642add275e5a8daa79f8f097751e9db2e8a.
